fast64_internal/oot/oot_actor.py

```diff
@@ -1,187 +1,529 @@
+        'TRIA_RIGHT')
+    
+    if headerItemProp.expandTab:
+        drawCollectionOps(box, index, propUser, None, objName)
+        prop_split(box, headerItemProp, 'headerIndex', 'Header Index')
+        if altProp is not None and headerItemProp.headerIndex >= len(altProp.cutsceneHeaders) + 4:
+            box.label(text = "Header does not exist.", icon = 'QUESTION')
 
+# Actor Header Property
 class OOTActorHeaderProperty(bpy.types.PropertyGroup):
-	sceneSetupPreset : bpy.props.EnumProperty(name = "Scene Setup Preset", items = ootEnumSceneSetupPreset, default = "All Scene Setups")
-	childDayHeader : bpy.props.BoolProperty(name = "Child Day Header", default = True)
-	childNightHeader : bpy.props.BoolProperty(name = "Child Night Header", default = True)
-	adultDayHeader : bpy.props.BoolProperty(name = "Adult Day Header", default = True)
-	adultNightHeader : bpy.props.BoolProperty(name = "Adult Night Header", default = True)
-	cutsceneHeaders : bpy.props.CollectionProperty(type = OOTActorHeaderItemProperty)
+    sceneSetupPreset : bpy.props.EnumProperty(name = "Scene Setup Preset", items = ootEnumSceneSetupPreset, default = "All Scene Setups")
+    childDayHeader : bpy.props.BoolProperty(name = "Child Day Header", default = True)
+    childNightHeader : bpy.props.BoolProperty(name = "Child Night Header", default = True)
+    adultDayHeader : bpy.props.BoolProperty(name = "Adult Day Header", default = True)
+    adultNightHeader : bpy.props.BoolProperty(name = "Adult Night Header", default = True)
+    cutsceneHeaders : bpy.props.CollectionProperty(type = OOTActorHeaderItemProperty)
 
 def drawActorHeaderProperty(layout, headerProp, propUser, altProp, objName):
-	headerSetup = layout.column()
-	#headerSetup.box().label(text = "Alternate Headers")
-	prop_split(headerSetup, headerProp, "sceneSetupPreset", "Scene Setup Preset")
-	if headerProp.sceneSetupPreset == "Custom":
-		headerSetupBox = headerSetup.column()
-		headerSetupBox.prop(headerProp, 'childDayHeader', text = "Child Day")
-		childNightRow = headerSetupBox.row()
-		childNightRow.prop(headerProp, 'childNightHeader', text = "Child Night")
-		adultDayRow = headerSetupBox.row()
-		adultDayRow.prop(headerProp, 'adultDayHeader', text = "Adult Day")
-		adultNightRow = headerSetupBox.row()
-		adultNightRow.prop(headerProp, 'adultNightHeader', text = "Adult Night")
-
-		childNightRow.enabled = not altProp.childNightHeader.usePreviousHeader if altProp is not None else True
-		adultDayRow.enabled = not altProp.adultDayHeader.usePreviousHeader if altProp is not None else True
-		adultNightRow.enabled = not altProp.adultNightHeader.usePreviousHeader if altProp is not None else True
-
-		for i in range(len(headerProp.cutsceneHeaders)):
-			drawActorHeaderItemProperty(headerSetup, propUser, headerProp.cutsceneHeaders[i], i, altProp, objName)
-		drawAddButton(headerSetup, len(headerProp.cutsceneHeaders), propUser, None, objName)
+    headerSetup = layout.column()
```

Comment:
So a sort of recurring theme that I am noticing is that your editor seems to like spaces instead of tabs. I would suggest trying to convert all of the spaces back to tabs since it is making there be tons of edits in this pr that I don't think you actually meant to change.

Reply:
Gonna switch back to tabs, I should have done that before PR-ing, my bad